libc/newhdrgen/yaml/math.yaml

```diff
@@ -2091,14 +2206,6 @@ functions:
       - type: _Float16
       - type: long
     guard: LIBC_TYPES_HAS_FLOAT16
-  - name: scalblnf16
-    standards:
-      - stdc
-    return_type: float16
-    arguments:
-      - type: float16
-      - type: long
-    guard: LIBC_TYPES_HAS_FLOAT16
```

Comment:
Should be reverted. 59f7a806182543c3d1198ad69c478afcc1443521 fixed the use of the internal `float16` type alias.